src/views/camp/camp.jsx

```diff
@@ -0,0 +1,137 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+
+require('./camp.scss');
+
+var Camp = injectIntl(React.createClass({
+    type: 'Camp',
+    render: function () {
+        return (
+            <div>
+                <TitleBanner className="masthead mod-blue-bg">
+                    <img src="/images/camp/ocean-top.svg" alt="Top" className="topImg"/>
+                    <div className="inner">
+                        <div className="title-content">
+                            <h1 className="title-banner-h1">
+                                <FormattedMessage id='camp.title' />
+                            </h1>
+                            <h4 className="intro title-banner-p">
+                                <FormattedMessage id='camp.dates' />
+                            </h4>
+                            <img src="/images/camp/bubbles.svg" alt="Bubbles" className="bubbles"/>
+                        </div>
+                    </div>
+                </TitleBanner>
+                <div className="gradient1">
+                    <section id="particpate">
+                        <div className="inner">
+                            <center>
+                                <h2><FormattedMessage id='camp.welcome' /></h2>
+                                <p id="intro">
+                                    <h2><FormattedMessage id='camp.welcomeIntro' /></h2>
```

Comment:
@St19Galla it looks like the text is still the same size on my browser:
<img width="1160" alt="screen shot 2017-07-18 at 9 03 37 am" src="https://user-images.githubusercontent.com/1815772/28318263-211feace-6b98-11e7-8890-006e4a3f43ff.png">

If I change `intro` to a `className="intro"` and remove the `h2` tags, it looks like this:
<img width="912" alt="screen shot 2017-07-18 at 9 06 28 am" src="https://user-images.githubusercontent.com/1815772/28318358-868679f0-6b98-11e7-9f92-d46414f4cdb6.png">
Which looks a bit more like the mocks. I personally think it looks better without the `h2`, but if you and @carljbowman like the `h2` then that's cool with me :).